Projects/b_u585i_iot02a_tfm/tfm.mk

```diff
@@ -61,8 +61,12 @@ NS_REGION_SIGNING_KEY ?= ${TFM_SRC_PATH}/bl2/ext/mcuboot/root-RSA-3072_1.pem
 ###############################################################################
 # Version number definition
 ###############################################################################
+APP_MAJOR_VERSION =  1
```

Comment:
This should be defined in project_defs.mk. NSPE_VERSION ?= "1.0.0" is meant as a default if project_defs.mk does not exist.

Reply:
I changed it to `APP_MAJOR_VERSION ?= 1` to keep value from project_defs.mk.